### PR TITLE
Update links to code signing docs

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>

    <tr>
      <td>:warning:</td>
      <td data-sticky="true"><code>mkdocs.yml</code> was modified - make sure no modification was made by mistake</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: mkdocs.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ site_description: "Documentation for fastlane tools, the easiest way to automate
 site_favicon: "img/favicon.ico"
 repo_url: "https://github.com/fastlane/docs"
 google_analytics: ["UA-18658848-13", "docs.fastlane.tools"]
-# strict: true # TODO: Enable once we have the docs structure
+strict: true
```
Comment (KrauseFx) [inline mkdocs.yml]: This in combination with https://github.com/fastlane/docs/pull/124, for some reason the flag is also needed

File: scripts/ci/generate_redirects.rb [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
   full_path = File.join(from, "index.html")
   to = "/#{to}" unless to.start_with?("/")
-  content = "<script type='text/javascript'>location.href='#{to}'</script>"
+  content = "<script type='text/javascript'>window.location.href = '#{to}#' + window.location.hash.substring(1);</script>"
```
Comment (mfurtak) [inline scripts/ci/generate_redirects.rb]: Why `substring` instead of taking the whole hash value?

**Comment by mfurtak** [comment]
👍

**Pull request merged**